apis/r/README.md

```diff
@@ -0,0 +1,3 @@
+# Needs to be
+
+typed up.
```

Comment:
@aaronwolen @eddelbuettel @mojaveazure I need your help with a skeleton here (which I'm happy to flesh out). Can you please confirm/deny/elaborate on the following:

* How to install
  * Note not from CRAN yet
  * How to install from RUniverse
  * How to install from `remotes::install_github`
  * How to install from `git checkout` and a source tree
* How to run tests
* More-info link for developers # Information for developers

Please see the [TileDB-SOMA wiki](https://github.com/single-cell-data/TileDB-SOMA/wiki).

Anything I'm missing here?

Reply:
Semi strong personal view (and a bit of an outlier):  `remotes::install_github` should be disouraged as it can lead to fetching 'random' state from the repo (though we are careful).  Releases are what makes the world go round reliably, and we have them -- and available at r-universe plus it is well documented there

![image](https://user-images.githubusercontent.com/673121/218331388-6a49ba47-8e5c-4658-b1e4-57aabe9ef5e9.png)

I would advise against overloading the repo with other information, if you feel strongly that you need to document how to check out the repo, install all dependencies and so on a vignette or wiki is in my view strongly preferable to overloading the README.md.

Reply:
@eddelbuettel I don't feel strongly about any of this and have no desire to overload anything -- I tossed out some ideas and am solicting feedback from R experts such as yourself. Thanks for giving me that! :)